target/mips/gdbstub.c

```diff
@@ -209,6 +210,14 @@ int mips_gdb_get_cheri_reg(CPUMIPSState *env, GByteArray *mem_buf, int n)
         }
         if (cheri_get_recent_pcc(env)->cr_tag)
             cap_valid |= ((uint64_t)1 << 32);
+        cap_valid |= ((uint64_t)env->active_tc.CHWR.UserTlsCap.cr_tag << 33);
```

Comment:
This is probably fine.  I think i had kind of left it this way on purpose to be compatible with what CheriBSD did as CheriBSD only supplies the GPRs in cap_valid.  I really need to figure out though how to send 129 bit registers across and remove cap_valid entirely.